pkg/computeclasses/computeclasses.go

```diff
@@ -121,11 +120,7 @@ func Validate(cc apiv1.ComputeClass, memory resource.Quantity, memDefault *int64
 	return nil
 }
 
-func CalculateCPU(cc internaladminv1.ProjectComputeClassInstance, memDefault *int64, memory resource.Quantity) (resource.Quantity, error) {
-	if err := ValidateProjectComputeClass(cc, memory, memDefault); err != nil {
-		return resource.Quantity{}, err
-	}
```

Comment:
Removed this check since it's being called with the memory value associated to the request and not the limit. The request value can be outside of the range of the min-max defined in the compute class, and can also be outside of the defined set of `values` on the compute class if configured